magicgui/type_map.py

```diff
@@ -43,6 +44,7 @@ class MissingWidget(RuntimeError):
 
 
 _RETURN_CALLBACKS: DefaultDict[type, list[ReturnCallback]] = defaultdict(list)
+_RETURN_MATCHERS: list[ReturnMatcher] = list()
```

Comment:
see comment in `types.py` ... let's just use `_RETURN_MATCHERS: list[TypeMatcher] = list()`

Reply:
Change implemented in 603ebd7